src/main/java/org/cascadebot/cascadebot/commands/fun/RandomNumberCommand.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.commands.fun;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.entities.Message;
+
+import org.cascadebot.cascadebot.UnicodeConstants;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.messaging.MessagingObjects;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import java.io.IOException;
+import java.util.Random;
+
+public class RandomNumberCommand implements ICommandMain {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        Random randomObj = new Random();
```

Comment:
Move this to a class object. no point making a new random every time we run this.
Actually @binaryoverload do you think we should have a global random for the entire bot?

Reply:
RandomUtils I think would be a good idea, @DeadlyFirex could you do this?

Reply:
:+1:

Reply:
Don't use this anymore, use the randomutils methods

Reply:
:+1: